schemas/2021-09-09/uiFormDefinition.schema.json

```diff
@@ -0,0 +1,3933 @@
+          "additionalProperties": true
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "kind",
+        "managementGroupId",
+        "location",
+        "parameters"
+      ]
+    },
+    "tenantScopeDeploymentContext": {
+      "type": "object",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "enum": [
+            "Tenant"
+          ]
+        },
+        "location": {
+          "type": "string"
+        },
+        "parameters": {
+          "type": "object",
+          "additionalProperties": true
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "kind",
+        "location",
+        "parameters"
+      ]
+    },
+    "stringResource": {
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "property": {
+              "type": "string"
+            },
+            "module": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "property"
+          ]
+        },
+        {
+          "type": "string"
+        }
+      ]
+    },
+    "formSteps": {
+      "type": "array",
+      "items": [
+        {
+          "$ref": "#/definitions/formStep"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        },
+        {
+          "$ref": "#/definitions/formStepWithoutResourceScope"
+        }
```

Comment:
I don't normally review UI schemas, but just happened to notice this, which seems odd. Do you know what's going on with the duplication here?